src/OrleansTestingHost/TestClusterOptions.cs

```diff
@@ -33,26 +83,41 @@ public TestClusterOptions()
         /// </summary>
         /// <param name="initialSilosCount">The number of initial silos to deploy.</param>
         public TestClusterOptions(short initialSilosCount)
+            : this(FallbackOptions.DefaultExtendedConfiguration)
         {
-            this.InitialSilosCount = initialSilosCount;
-            this.BaseSiloPort = ThreadSafeRandom.Next(22300, 30000);
-            this.BaseGatewayPort = ThreadSafeRandom.Next(40000, 50000);
+            this.ExtendedFallbackOptions.InitialSilosCount = initialSilosCount;
         }
 
+
         /// <summary>
-        /// The default initial silos count. See <see cref="TestClusterOptions"/>.
+        /// Create a new TestClusterOptions
```

Comment:
Not related to this PR, but if we decide on StyleCop settings, then there's usually a set [constructor doc string](http://stylecop.soyuz5.com/SA1642.html)